File: cfgs/pythoncfgs/NUMBER_IS_DIVISIBLE_BY_29_OR_NOT.py
Nodes
1;conditional;fun_1(foo_1 / 100);8;
2;statement;return foo_1 % 29 == 0;12;
3;statement;foo_1 = fun_1(foo_2 % 10);9;
4;statement;foo_1 = fun_1(foo_2 / 10);10;
5;statement;foo_1 += foo_2 * 3;11;
6;exit;;
Edges
0;2;3;
1;6;
2;4;
3;5;
4;1;
5;

File: cfgs/java_cfgs_foo/NUMBER_IS_DIVISIBLE_BY_29_OR_NOT.java
Nodes
1;special;;;
2;conditional;(foo_1 / 100) > 0;14;
3;statement;int last_digit = ((int) (foo_1)) % 10;15;
4;statement;n /= 10;16;
5;statement;n += foo_1 * 3;17;
6;statement;return (foo_1 % 29) == 0;19;
7;exit;;;
Edges
0;2;
1;6;3;
2;4;
3;5;
4;2;
5;7;
6;
